terraform/variables.tf

```diff
@@ -61,12 +61,6 @@ variable "kv_certificate_subject" {
   type        = string
 }
 
-variable "asp_sku" {
-  default     = "S1"
-  description = "SKU name for the App Service Plan"
-  type        = string
-}
-
```

Comment:
I think this one will still be needed otherwise we won't be able to pass in the sku?

Reply:
shall I just remove the default so make it less confusing as to where is the true value?